projects/common/src/index.ts

```diff
@@ -428,6 +428,8 @@ export interface TrailImage extends Image {
 
 export interface CreditedImage extends Image {
     credit?: string
+    caption?: string
+    displayCredit?: boolean
```

Comment:
Worth noting that, in time, I expect to be able to take the question marks from some of these fields but currently we don't always get a value from the tooling.